### DISCUSSION TITLE
Sorting table in front-end composer

### DISCUSSION THREAD

**Discussion by rmuller-vu** [mia-platform/community#552] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

I am trying to use the response from an http event to send information to a custom component. I have added the `triggeredBy` property but the data is null. In the below example when confirm is selected on the require modal it makes an http GET request to `https://someurl/route` and receives back a payload. I added the `triggeredBy` property with the value of `data_1`. When I try to use the `data_1` in the `onSuccess` event.

```json
{
  "tag": "bk-button",
  "properties": {
    "content": "Diff Versions",
    "bulkButton": true,
    "action": {
      "type": "event",
      "config": {
        "events": {
          "label": "require-confirm",
          "payload": {
            "title": "{{#if selectedData.[1]}}Confirm?{{else}}Please Review!{{/if}}",
            "content": "{{#if selectedData.[1]}}Diff source {{selectedData.[0].versionName}} and target {{selectedData.[1].versionName}}{{/if}}",
            "configOk": {
              "tag": "bk-button",
              "properties": {
                "content": "Confirm",
                "action": {
                  "type": "http",
                  "config": {
                    "url": "https://someurl/route",
                    "method": "GET",
                    "triggeredBy": "data_1",
                    "body": []
                  },
                  "hooks": {
                    "onSuccess": {
                      "type": "event",
                      "config": {
                        "events": {
                          "label": "open-diff-modal",
                          "payload": {
                            "responseData": "data_1"
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
          }
        }
      }
    }
  }
}
```

**Comment by LiliumCandidum** [comment]
Hi @rmuller-vu,

I'm not sure I'm understanding what you would like to do and what are you expecting. Can you add more details to help you in the best way?

**Reply by rmuller-vu** [reply]
I am trying to use `triggeredBy` but my example isn't working. This is from the docs example but `data_1` for me is null. How do I properly use `triggeredBy` to get the response data from the initial http get call

**Reply by LiliumCandidum** [reply]
Ok, I see. I think you should use the handlebars in your `open-diff-modal` event:

```
"events": {
    "label": "open-diff-modal",
    "payload": {
      "responseData": "{{rawObject data_1}}"
    }
```

I added `rawObject` because I assumed the response is an object.
